### DISCUSSION TITLE
assigning domain suddenly takes long time

### DISCUSSION THREAD

**Discussion by dlibis** [vercel/community#4595] [Bug Report]
### Page to Investigate

n/a

### Steps to Reproduce

Hey,
usually that part takes a few seconds, but now its already a couple of minutes, see image
![image](https://github.com/vercel/community/assets/63005208/12f62cae-d12e-4aff-9297-051c1dc35225)

**Comment by yannJaster10i9** [comment]
same issue here

**Comment by gregfromstl** [comment]
Same issue here, started happening about 30 minutes ago. The whole site is extremely slow as well

**Comment by pkage** [comment]
Same here. I think it's on their end.

**Comment by amyegan** [comment]
Thanks for reporting this! The team is already aware and looking into the problem. You can follow along in https://github.com/vercel/community/discussions/4594 and https://www.vercel-status.com/incidents/ndnw1mxqjkh8 for the latest updates 🙇

**Reply by dlibis** [reply]
i can confirm now its resolved, and takes 1 sec
